dojo/tools/gitlab_secret_detection_report/parser.py

```diff
@@ -0,0 +1,64 @@
+import json
+from dojo.models import Finding
+
+
+class GitlabSecretDetectionReportParser(object):
+    """
+    GitLab's secret detection report
+    See more: https://gitlab.com/gitlab-org/security-products/security-report-schemas/-/blob/master/dist/secret-detection-report-format.json
+    """
+
+    def get_scan_types(self):
+        return ["GitLab Secret Detection Report"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "GitLab Secret Detection Report"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "GitLab Secret Detection Report file can be imported in JSON format (option --json)."
+
+    def get_findings(self, file, test):
+        # Load JSON data from uploaded file
+        data = json.load(file)
+
+        findings = []
+
+        # This is required by schema - it won't be null / undefined
+        date = data["scan"]["end_time"]
```

Comment:
@natebwangsut In #4808 you implemented a bugfix for a similar line of code. Do we need the same here?

Reply:
@StefanFl yup, i've also fixed them here as well